utils/securityJobSummary.go

```diff
@@ -187,7 +187,7 @@ func RecordSarifOutput(cmdResults *Results) (err error) {
 		log.Info("Results can be uploaded to Github security tab automatically by upgrading your JFrog subscription.")
 		return
 	}
-	sarifReport, err := GenerateSarifReportFromResults(cmdResults, true, false, nil)
+	sarifReport, err := GenerateSarifReportFromResults(cmdResults, true, false, nil, GetAllSupportedScans(), cmdResults.ResultType)
```

Comment:
I would pass the list to the `RecordSarifOutput` func as arg, so all the places that uses it can control and pass the values if they have

Reply:
done